recipe/conda_build_config.yaml

```
@@ -1,3 +1,6 @@
-libstdcxx:  # [linux]
-  - 13      # [linux]
-  - 14      # [linux]
+python_variants:
```

I think the naming convention is to just use the package name directly

```suggestion
- python:
```

and then the same naming convention in the host jinja template

```
host:
  - python {{ python }}
```

---

conda documentation uses a simple ``python`` for the python variant: https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#build-variants

---

I see some conda-forge recipes using the same python variants: https://github.com/search?type=code&q=org%3Aconda-forge+path%3Arecipe%2Fconda_build_config.yaml+python%3A.

It's a surprisingly small number and I don't recognize many package names, but between that list and the fact that this solution has been working internally I'm pretty confident that this explicit list of Python variant versions is what we're looking for.

---

resolved by no longer applying